components/coda/actions/copy-doc/copy-doc.mjs

```diff
@@ -0,0 +1,41 @@
+import coda from "../../coda.app.mjs";
+
+export default {
+  key: "coda_copy-doc",
```

Comment:
Use `coda-` instead of `coda_`, so this should be `coda-copy-doc`.

We use underscores in place of spaces when the name of the app itself includes them, like in `google_drive-copy-doc`

Reply:
Issue addressed